website/api/zkvm/zkvm-specification.md

```diff
@@ -35,22 +35,49 @@ preemption, so there is also no support for atomic instructions. If the
 execution raises an exception such as an unaligned access, the execution
 terminates without executing exception handlers.
 
+## The User/Kernel Split
+
+Starting with the v2 circuit, the RISC Zero zkVM introduces distinct user and
```

Comment:
I feel like 'v2 circuit' makes sense to us but not necessarily to the outside world.

Reply:
I removed the mention